iroh-net/src/relay/http/client.rs

```diff
@@ -275,6 +291,18 @@ impl ClientBuilder {
         self
     }
 
+    /// Set an explicit proxy url to proxy all HTTP(S) traffic through.
+    pub fn proxy_url(mut self, url: Url) -> Self {
+        self.proxy_url.replace(url);
+        self
+    }
+
+    /// Disable http proxy entirely, no matter the environment variables.
+    pub fn no_proxy(mut self) -> Self {
+        self.no_proxy = true;
+        self
+    }
```

Comment:
This (and the field) are unused?  Left over from a previous version?

Reply:
removed